js/abc.js

```diff
@@ -11,7 +11,7 @@
 
 // This header is prepended to the Abc output.
 const ABCHEADER = "X:1\nT:Music Blocks composition\nC:Mr. Mouse\nL:1/16\nM:C\n";
-
+let  notes, note;
```

Comment:
Why do you have this here instead of near Line 132?

Reply:
it was declared within one scope, and used outside that scope, it worked okay, because of `var`.